components/script/dom/bindings/str.rs

```diff
@@ -671,3 +675,16 @@ fn max_week_in_year(year: u32) -> u32 {
 fn is_leap_year(year: u32) -> bool {
     year % 400 == 0 || (year % 4 == 0 && year % 100 != 0)
 }
+
+// https://html.spec.whatwg.org/multipage/#rules-for-parsing-floating-point-number-values
```

Comment:
Maybe this one could be a documentation comment which should be start with `///`.

Reply:
good catch